ansible/roles/test/files/ptftests/vxlan-decap.py

```diff
@@ -343,15 +403,19 @@ def checkRegularRegularLAGtoVLAN(self, acc_port, net_port, test):
         self.dataplane.flush()
         for i in xrange(self.nr):
             testutils.send_packet(self, net_port, packet)
-        nr_rcvd = testutils.count_matched_packets(self, exp_packet, acc_port, timeout=0.5)
+        # We don't care if expected packet is received during warming up
+        if not wu:
+            nr_rcvd = count_matched_packets_helper(self, exp_packet, self.nr, acc_port, timeout=20)
+        else:
+            nr_rcvd = 0
```

Comment:
but you don't use this function in the warmup at all
Why do we need this logic here?

Reply:
checkRegularRegularLAGtoVLAN is called from RegularLAGtoVLAN, and used in warmup.